operator/deploy/crds/csiscaleoperators.csi.ibm.com_cr.yaml

```diff
@@ -11,29 +11,29 @@ metadata:
     release: ibm-spectrum-scale-csi-operator
 status: {}
 spec:
-  # The path to the GPFS file system mounted on the host machine.
-  # ==================================================================================
+# The path to the GPFS file system mounted on the host machine.
```

Comment:
Can we change description for GPFS file system hostpath for better understanding

`# The path to the GPFS file system mounted (either remote/local) on the local Spectrum Scale API host machine.`